src/controllers/appointment.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from 'express';
+import { Op } from 'sequelize';
+import Appointment from '../models/appointment';
+import sequelize from '../db/connection';
+
+
+const getAppointments = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const { date } = req.query;
+  const { therpaistId } = req.params;
+  try {
+    const query = await Appointment.findAll({
+      attributes: ['id', 'datetime', 'therapistId'],
+      where: {
+        [Op.and]: [

```

Comment:
why to have op.and,
we can do this in more readable way

```suggestion
      where: {
        therapistId: ,
        datetime: ,
        }

```

Reply:
I tried it, it doesn't work, I'm using a date function to match date Query with datetime db. col, and it doesn't allow me to assign this as a value for datetime key.